dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/SiteRestRepository.java

```diff
@@ -44,10 +43,10 @@ public SiteRestRepository() {
     }
 
     @Override
-    public SiteRest findOne(Context context, UUID id) {
+    public SiteRest findOne(UUID id) {
```

Comment:
Since one and only one site object will exist, should there be another method to pull the single site object without specifying the id?

Reply:
Is the `Site` object actually used in DSpace 6?